src/server/database/Database/Implementation/LoginDatabase.cpp

```diff
@@ -87,7 +87,7 @@ void LoginDatabaseConnection::DoPrepareStatements()
     PrepareStatement(LOGIN_GET_USERNAME_BY_ID, "SELECT username FROM account WHERE id = ?", CONNECTION_SYNCH);
     PrepareStatement(LOGIN_SEL_CHECK_PASSWORD, "SELECT 1 FROM account WHERE id = ? AND sha_pass_hash = ?", CONNECTION_SYNCH);
     PrepareStatement(LOGIN_SEL_CHECK_PASSWORD_BY_NAME, "SELECT 1 FROM account WHERE username = ? AND sha_pass_hash = ?", CONNECTION_SYNCH);
-    PrepareStatement(LOGIN_SEL_PINFO, "SELECT a.username, aa.gmlevel, a.email, a.reg_mail, a.last_ip, DATE_FORMAT(a.last_login, '%Y-%m-%d %T'), a.mutetime, a.mutereason, a.muteby, a.failed_logins, a.locked, a.OS FROM account a LEFT JOIN account_access aa ON (a.id = aa.id AND (aa.RealmID = ? OR aa.RealmID = -1)) WHERE a.id = ?", CONNECTION_SYNCH);
+    PrepareStatement(LOGIN_SEL_PINFO, "SELECT a1.username, a2.gmlevel, a1.email, a1.reg_mail, a1.last_ip, DATE_FORMAT(a1.last_login, '%Y-%m-%d %T'), a3.mutetime, a3.mutereason, a3.mutedby, a1.failed_logins, a1.locked, a1.os FROM account AS a1 LEFT JOIN account_access AS a2 ON (a1.id = a2.id AND (a2.RealmID = ? OR a2.RealmID = -1)) LEFT JOIN account_muted AS a3 ON (a1.id = a3.account AND (a3.active = 1)) WHERE a1.id = ?", CONNECTION_SYNCH);
```

Comment:
it's kinda unreadable with account AS a1, account_access AS a2 and account_muted AS a3